### ISSUE TITLE
One-Step Migration not working from Magento CE 1.8.10

### ISSUE THREAD

**Issue by agentmedia** [OpenMage/magento-lts#3390]
### Preconditions (*)

1. Having a running OpenMage CE1.8.10 Installation
2. Switching to PHP 7.4
3. Attempt to migrate using one step migration

### Steps to reproduce (*)

1. Magento CE 1.8.1.0 installed
2. Run curl -fsSL https://migrate.openmage.org | sh

### Expected result (*)

1. Should update codes and database to OpenMage LTS v19.4.8

### Actual result (*)

Script runs into the following error. When trying to call the mentioned the s3 bucket url, it says this bucket does not exist.

Migration will begin in 20 seconds.
Press CTRL+C to abort.

Detected that you have installed Magento Commuinity Edition 1.8.1.0
Downloading patch for Magento CE 1.8.1.0 to OpenMage LTS v19.4.8...
--2023-07-18 17:35:15--  https://openmage.migrationpatches.s3.amazonaws.com/magento-ce-1.8.1.0-openmage-lts-v19.4.8.patch.gz
Resolving openmage.migrationpatches.s3.amazonaws.com (openmage.migrationpatches.s3.amazonaws.com)... ::ffff:3.5.8.196, ::ffff:52.216.1.144, ::ffff:52.216.38.233, ...
Connecting to openmage.migrationpatches.s3.amazonaws.com (openmage.migrationpatches.s3.amazonaws.com)|::ffff:3.5.8.196|:443... connected.
The certificate's owner does not match hostname ‘openmage.migrationpatches.s3.amazonaws.com’
HTTP request sent, awaiting response... 404 Not Found
2023-07-18 17:35:15 ERROR 404: Not Found.

**Comment by agentmedia** [comment]
Additional Information: Switched to PHP 7.4 before running OpenMage migration script.

**Comment by pquerner** [comment]
Duplicate of #1415, #3361, #3343

tldr: there is no migration script

**Comment by ADDISON74** [comment]
This report is not a bug in OpenMage. In the past any bug had to be reported here https://github.com/OpenMage/migrate, but whoever visits this page may find that it is no longer valid.

Why wasn't the repo deleted? I cannot give a ckear answer, maybe in time someone will clone the repository and update the migration script. But we have much simpler and safer methods of updating OpenMage right now.

**Comment by fballiano** [comment]
the report has a huge "discontinued" disclaimer: https://github.com/OpenMage/migrate

problem is nobody is updating the website

**Comment by fballiano** [comment]
https://www.openmage.org/magento-lts/migration-guide.html
